js/src/components/Suggestion.js

```diff
@@ -25,8 +26,11 @@ class Suggestion extends React.Component {
 					<RaisedURLNewWindowButton { ...this.props.button } { ...buttonProps } />
 				</div>
 				<div className="yoast-wizard--column__push_left yoast-wizard--video-frame">
-					<iframe width="400" height="225" src={ this.props.video } frameBorder="0"
-							allowFullScreen/>
+					<iframe
+						width="400" height="225" src={ this.props.video } frameBorder="0"
+						allowFullScreen={ true }
+						title={ __( "Video tutorial", "wordpress-seo" ) }
```

Comment:
Ok so this is causing the missing domain error. This string should be passed as a prop from the config.